src/test/java/org/takes/rq/RqMultipartTest.java

```diff
@@ -79,6 +80,23 @@ public void satisfiesEqualsContract() throws IOException {
             ),
             "Content-Disposition: form-data; name=\"data\"; filename=\"a.bin\""
         );
+        final String boundary = "AaB02x";
```

Comment:
@erimerturk I still think that you can use `RqMultipart.Fake`.. why not? `RqMultipart.Fake` implements `Request`